RFS/src/main/java/com/rfs/cms/OpenSearchWorkCoordinator.java

```diff
@@ -516,42 +533,26 @@ UpdateResult assignOneWorkItem(long expirationWindowSeconds) throws IOException
             + "\"size\": 1,\n"
             + "\"script\": {"
             + "  \"params\": { \n"
-            + "    \"clientTimestamp\": "
-            + CLIENT_TIMESTAMP_TEMPLATE
-            + ",\n"
-            + "    \"expirationWindow\": "
-            + EXPIRATION_WINDOW_TEMPLATE
-            + ",\n"
-            + "    \"workerId\": \""
-            + WORKER_ID_TEMPLATE
-            + "\",\n"
+            + "    \"clientTimestamp\": " + CLIENT_TIMESTAMP_TEMPLATE + ",\n"
+            + "    \"expirationWindow\": " + EXPIRATION_WINDOW_TEMPLATE + ",\n"
+            + "    \"workerId\": \"" + WORKER_ID_TEMPLATE + "\",\n"
             + "    \"counter\": 0\n"
             + "  },\n"
             + "  \"source\": \""
-            + "      if (ctx._source.scriptVersion != \\\""
-            + SCRIPT_VERSION_TEMPLATE
-            + "\\\") {"
+            + "      if (ctx._source.scriptVersion != \\\"" + SCRIPT_VERSION_TEMPLATE + "\\\") {"
             + "        throw new IllegalArgumentException(\\\"scriptVersion mismatch.  Not all participants are using the same script: sourceVersion=\\\" + ctx.source.scriptVersion);"
             + "      } "
             + "      long serverTimeSeconds = System.currentTimeMillis() / 1000;"
             + "      if (Math.abs(params.clientTimestamp - serverTimeSeconds) > {CLOCK_DEVIATION_SECONDS_THRESHOLD}) {"
             + "        throw new IllegalArgumentException(\\\"The current times indicated between the client and server are too different.\\\");"
             + "      }"
             + "      long newExpiration = params.clientTimestamp + (((long)Math.pow(2, ctx._source.numAttempts)) * params.expirationWindow);"
-            + "      if (ctx._source."
-            + EXPIRATION_FIELD_NAME
-            + " < serverTimeSeconds && "
+            + "      if (ctx._source." + EXPIRATION_FIELD_NAME + " < serverTimeSeconds && "
             + // is expired
-            "          ctx._source."
-            + EXPIRATION_FIELD_NAME
-            + " < newExpiration) {"
+            "          ctx._source." + EXPIRATION_FIELD_NAME + " < newExpiration) {"
             +      // sanity check
-            "        ctx._source."
-            + EXPIRATION_FIELD_NAME
-            + " = newExpiration;"
-            + "        ctx._source."
-            + LEASE_HOLDER_ID_FIELD_NAME
-            + " = params.workerId;"
+            "        ctx._source." + EXPIRATION_FIELD_NAME + " = newExpiration;"
+            + "        ctx._source." + LEASE_HOLDER_ID_FIELD_NAME + " = params.workerId;"
```

Comment:
I'm putting an info() message alongside "Running RfsWorker" (now "Running RfsMigrateDocuments with workerId = ...").  I've also added the value to the MDC.  The WorkCoordinatorErrantAcquisitonsRetryTest sets the MDC value as well and the DocumentFromSnapshotMigration test log4j2.properties file shows how log4j2 can be configured to print it on every logged line (or as in the test file, every log statement for the OpenSearchWorkCoordinator).  I'm hesitant to change the log4j2 properties around for the main program because it doesn't have one.  Instead the RFS package does and I'm not sure how we're loading the log4j2 configs today for the RFS derivative packages.

Reply:
Thanks!